src/containers/Review/assignment/AssignmentSectionRow.tsx

```diff
@@ -135,17 +134,17 @@ const AssignmentSectionRow: React.FC<AssignmentSectionRowProps> = ({
               setIsReassignment={setIsReassignment}
               setIsUnassignment={() => showModal({ onConfirm: () => unassignAssignee() })}
             />
+          ) : assignmentOptions.options.length === 0 ? (
+            <Label className="simple-label" content={strings.ASSIGNMENT_NOT_AVAILABLE} />
```

Comment:
Another tiny improvement to show in AssignmentTab the label "Assignment not available" when there is none - but the user can still see the Assignment page because they can Self-assign. But currently it is assigned to another user.